homeassistant/components/rtsp_to_webrtc/__init__.py

```diff
@@ -67,7 +67,7 @@ async def async_offer_for_stream_source(
         """
         try:
             async with async_timeout.timeout(TIMEOUT):
-                return await client.offer_stream_id(offer_sdp, stream_source, stream_id)
+                return await client.offer_stream_id(stream_id, offer_sdp, stream_source)
```

Comment:
For reviewer: This is the signature being called: https://github.com/allenporter/rtsp-to-webrtc-client/blob/197038454f7f6c8ccc35b161249fd8fb2114f9d2/rtsp_to_webrtc/webrtc_client.py#L38